packages/core/src/components/forms/numericInput.tsx

```diff
@@ -393,6 +447,26 @@ export class NumericInput extends AbstractPureComponent<HTMLInputProps & INumeri
         }
     };
 
+    private startContinuousChange() {
+        // The button's onMouseUp event handler doesn't fire if the user
+        // releases outside of the button, so we need to watch all the way
+        // from the top.
+        document.addEventListener("mouseup", this.handleMouseUp);
+        this.setTimeout(() => {
```

Comment:
why wrap this in a timeout? if it's necessary, leave a code comment explaining why.
seems like just setting the interval directly should do it.

Reply:
There's an initial delay that's slightly longer (currently 300ms) than the regular interval that follows (currently 100ms); it's so that users don't trigger the continuous increment by accident. AFAIK it's not possible to change the interval of an existing `setInterval()` call. Does that make sense?

Reply:
ohhh yeah i see that. def leave a comment in the code.